app/components/CallCharacter/index.tsx

```diff
@@ -263,14 +264,22 @@ export function CallCharacter({ character }: { character: CharacterType }) {
         state: state,
       });
     };
-    session.onError = () => {
+    session.onError = (err: VoiceSessionError) => {
+      const msg = err.message;
       console.log(
-        `CallCharacter[${session.conversationId}]: voiceSession error`
+        `CallCharacter[${session.conversationId}]: voiceSession error: ${msg}`
       );
       track("voice-session-error", {
         conversationId: session.conversationId || "",
+        error: msg,
       });
-      session.stop();
+      // We have a funky situation here where 1006 errors sometimes are due to React Strict Mode
+      // causing the VoiceSession to get re-created, which we don't want to barf on, but sometimes
+      // are due to a server-side error that occurs during the websocket connection startup. For
+      // now, we ignore them but still log them.
+      if (!msg.indexOf("1006")) {
```

Comment:
This might still be OK though since we're fine with stopping the session that got killed due to strict mode?